ecs-cli/modules/cli/local/secrets/clients/clients.go

```diff
@@ -68,7 +72,7 @@ func (d *SSMDecrypter) DecryptSecret(arnOrName string) (string, error) {
 		WithDecryption: aws.Bool(true),
 	})
 	if err != nil {
-		return "", errors.Wrapf(err, "Failed to retrieve decrypted secret from %s due to %v", arnOrName, err)
+		return "", errors.Wrapf(err, "failed to retrieve decrypted secret from %s due to %v", arnOrName, err)
```

Comment:
Like David mentioned yesterday, do you think it would be clearer to put the error message in a new line.